app/src/main/java/com/myperishableplanner/v21001/MainActivity.kt

```diff
@@ -46,7 +46,7 @@ class MainActivity : ComponentActivity() {
 
     private var firebaseUser: FirebaseUser? = FirebaseAuth.getInstance().currentUser
     private var selectedItem: Item ? = null
-    private var selectedCategory : Category? = null
+    private var selectedCategory: Category? = null
```

Comment:
Proper coding convention states not to put a space before the colon when used for a declaration and its type.